features/screen-capture.yml

```
@@ -1,7 +1,8 @@
-draft_date: 2024-10-28
-name: Screen Capture
-description: TODO
+name: Screen capture
+description: The `navigator.mediaDevices.getDisplayMedia()` method lets the user select a screen or portion of a screen (such as a window) to capture as a media stream.
```

I think the fact that this actively prompts the user is useful to note:

```suggestion
description: The `navigator.mediaDevices.getDisplayMedia()` method asks the user to choose a screen or portion of a screen (such as a window) to capture as a media stream.
```